docs/user/using-an-OIDC-provider.md

```diff
@@ -49,7 +49,86 @@ For Dex, you can find the parameters that you need to set in the configuration (
 
 ### Azure Active Directory
 
-For setting up an Azure Kubernetes cluster (aks) with Azure Active Directory you can follow [this guide](https://docs.microsoft.com/en-us/azure/aks/aad-integration). At the end of the tutorial you should have an Active Directory Application (Server). That's the Application from which we will get the needed parameters.
+#### AKS-managed Azure Active Directory
+
+This setup assumes that you are using the [AKS-managed Azure AD integration](https://docs.microsoft.com/en-us/azure/aks/managed-aad). Note that this is incompatible with the legacy AAD. We will use the v2 token version instead of v1.
+
+> If you really need to use v1 tokens, please let drop us an issue and let us know your use case.
+
+Please refer to the official docs in case of doubt, here we just highlight some important steps during the configuration that will be required in Kubeapps.
+
+First, you need to verify that you have `AKS-managed Azure Active Directory: Enabled`.
+
+![Enabled AKS-managed Azure Active Directory](../img/azure-00.png)
+
+Then, you have to register an application. Go to `App registrations and click on `Register an application. Enter a `Name` (any name is ok) and a `Redirect URI` (it should be `https://<your_kubeapps_domain/>oauth2/callback` unless you have manually changed it).
+
+![Registering an application](../img/azure-01.png)
+
+Once you got your application created, click `Overview` to see the `Application (client) ID` and the `Directory (tenant) ID`. You will need these values later.
+
+![Application overview](../img/azure-02.png)
+
+The next step is to create a secret for the application. Click on `Certificates & secrets` and then on `New client secret`.
+
+![Creating secret screen](../img/azure-03.png)
+
+Fill the `Description` and `Expires` fields according to your preferences.
+
+![Creating the secret](../img/azure-04.png)
+
+Next, the `Value` of the secret will appear and you will be able to copy it. You will need this value later.
+
+![Retrieving the secret value](../img/azure-05.png)
+
+The following step is to define the permissions that the application will need. To do so, go to `API permissions` and click `Add a permission`.
+
+![Add permissions screen](../img/azure-06.png)
+
+Then, in the `APIs my organization uses` enter this value `6dae42f8-4368-4678-94ff-3960e28e3630` (this is the ID corresponding to the AKS AAD server).
```

Comment:
```suggestion
Then, in the `APIs my organization uses` enter this value `6dae42f8-4368-4678-94ff-3960e28e3630`. This is a special ID corresponding to the global Azure Kubernetes Service AAD server.
```
I don't know if "global" is correct here, but assuming it's a constant value for everyone (all AKS accounts), I assume so. But ideally it'd be great to link to some documentation about this ID, even if it's just weinong's docs.

Reply:
Good suggestion! Well, this value is a way to associate which subscription an Azure resource (AKS, AAD or anything else) belongs to. For instance, if using `common` as the tenant ID (in theory) you will open the login to any Microsoft account, but I haven't explored it.
Yep, I'll link the [azure docs](https://docs.microsoft.com/en-us/azure/active-directory/fundamentals/active-directory-how-to-find-tenant) here. Thank you!